storage/mroonga/vendor/groonga/CMakeLists.txt

```diff
@@ -242,6 +242,7 @@ ac_check_lib(dl dlopen)
 ac_check_lib(execinfo backtrace)
 if(HAVE_LIBEXECINFO)
   set(HAVE_BACKTRACE TRUE)
+  set(HAVE_EXECINFO_H TRUE)
 else()
```

Comment:
Why is that needed? HAVE_EXECINFO_H is checked and set in `configure.cmake`:
```
CHECK_INCLUDE_FILES (execinfo.h HAVE_EXECINFO_H)
```

Reply:
Yes I knew but it did not work. Usually find_path works better (backtrace needs an external library and is not in system folders neither in BSD unlike Linux). But I m not against removing it.